nx/lib/nx/lin_alg.ex

```diff
@@ -499,7 +499,7 @@ defmodule Nx.LinAlg do
 
     output_type = a_type |> Nx.Type.merge(b_type) |> Nx.Type.to_floating()
 
-    output = %T{b | shape: output_shape, type: output_type, names: nil}
+    output = %T{b | shape: output_shape, type: output_type, names: List.duplicate(nil, tuple_size(output_shape))}
```

Comment:
To make the formatter happy:

```suggestion
    output_names = List.duplicate(nil, tuple_size(output_shape))
    output = %T{b | shape: output_shape, type: output_type, names: output_names}
```